src/api/app/views/webui/package/binary.html.haml

```diff
@@ -0,0 +1,54 @@
+- @pagetitle = "Detailed Information About #{@filename}"
+- @metarobots = 'noindex' # files change too often
+- package_bread_crumb(link_to('Binaries', package_binaries_path(project: @project,
+    package: @package, repository: @repository.name, arch: @arch.name)), truncate(@filename, length: 30))
+
+= render partial: 'tabs'
```

Comment:
missing `locals` in tabs

Reply:
It is still the old one :smile:

Reply:
I know, but this is not needed here since it's for the old UI:laughing: